samples/99_zil_interpreter/app/builtins.rb

```diff
@@ -133,6 +209,7 @@ def define_for_evaled_arguments(&implementation)
   result
 }
 
+#! shouldn't those just be doable with `Enumerable#all?`
```

Comment:
Technically it needs to return the actual value of the last non-falsey thing but yet it could still be shorter like
```rb
arguments.all? && arguments.last
```

But anyways this and the comment below can be addressed in another PR ;) - so maybe let's remove them for now and fix later